pkg/apiserver/info/info.go

```diff
@@ -38,15 +38,16 @@ func NewService(config *config.Config, tidbForwarder *tidb.Forwarder, db *dbstor
 
 func Register(r *gin.RouterGroup, auth *user.AuthService, s *Service) {
 	endpoint := r.Group("/info")
-	endpoint.Use(auth.MWAuthRequired())
 	endpoint.GET("/info", s.infoHandler)
```

Comment:
`/info` API should not be auth required, else it has no normal response before login. And I think it is fine to non-auth-required.